pypfopt/efficient_semivariance.py

```diff
@@ -0,0 +1,134 @@
+"""
+The ``efficient_semivariance`` module houses the EfficientSemivariance class, which
+generates optimal portfolios with respect to the semivariance of the portfolio.
+"""
+
+import warnings
+
+import cvxpy as cp
+import numpy as np
+import pandas as pd
+
+from . import base_optimizer, objective_functions
+
+
+class EfficientSemivariance(base_optimizer.BaseConvexOptimizer):
+
+    # Note: expected_returns and historic_returns have to be in the same frequency
+    def __init__(
+        self,
+        expected_returns,
+        historic_returns,
```

Comment:
Compared to the other optimizers, semivariance optimization requires the complete historic return matrix. However, it must be ensured that the expected returns and the historic returns are in the same frequency. I think this is rather hard to ensure, as one needs to specify the frequency explicitly when `expected_returns` are estimated (see tests).